pkg/pillar/hypervisor/acrn.go

```diff
@@ -45,8 +45,8 @@ func (ctx acrnContext) Delete(domainName string, domainID int) error {
 }
 
 // Info outputs domain info via logging
-func (ctx acrnContext) Info(domainName string, domainID int) (int, DomState, error) {
-	return 0, Running, nil
+func (ctx acrnContext) Info(domainName string, domainID int) (int, types.SwState, error) {
+	return 0, types.RUNNING, nil
```

Comment:
What is the value of putting this in `types/`?

The `types/` package is getting massively overloaded. This should either be exported types in `hypervisor/` like before (what was wrong with that?), or if it has to be elsewhere for some reason, then in something reasonable dedicated like `types/hypervisor/`, but then why not just keep it here?

Reply:
The logic here being that the domains are slowly being replaced by tasks (where a task has a domain behind it is immaterial). Thus a common typesystem of a states a task (FKA domain) can be in is expected to be located in a single place. For now that place is domainmgrtypes.go